plugins/woocommerce-blocks/assets/js/blocks/product-filter/inner-blocks/price-filter/frontend.ts

```diff
@@ -126,5 +194,40 @@ store< PriceFilterStore >( 'woocommerce/product-filter-price', {
 				} )
 			);
 		},
+		updateRange: ( event: HTMLElementEvent< HTMLInputElement > ) => {
+			const context = getContext< PriceFilterContext >();
+			const { minPrice, maxPrice, minRange, maxRange } = context;
+			const isMin = event.target.classList.contains( 'min' );
+			const targetValue = +event.target.value;
```

Comment:
The "plus" in front of the `+event.target.value` is being used uniquely to convert it to a number?

In that case, wouldn't it be better to make this more explicitly? For instance using `Number`:

```suggestion
			const targetValue = Number(event.target.value);
```

or using `parseFloat` / `parseInt`:

```suggestion
			const targetValue = parseFloat(event.target.value);
```